app/code/Magento/Inventory/Test/Acceptance/Test/AdminPaginationWorkedInManageSourcesGridTest.xml

```diff
@@ -0,0 +1,76 @@
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource6"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource7"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource8"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource9"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource10"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource11"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource12"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource13"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource14"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource15"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource16"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource17"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource18"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource19"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource20"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource21"/>
+            <createData entity="SpecialSourceForAdminPaginationWorkedInManageSourcesGridTest" stepKey="createSource22"/>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin1"/>
+        </before>
+
+        <after>
+            <actionGroup ref="logout" stepKey="logoutOfAdmin1"/>
+        </after>
+
+        <amOnPage url="{{AdminManageSourcePage.url}}" stepKey="amOnTheManageSourcePage1"/>
+        <waitForPageLoad time="30" stepKey="waitForPageLoad1"/>
+        <conditionalClick selector="{{AdminDataGridHeaderSection.clearFilters}}" dependentSelector="{{AdminDataGridHeaderSection.clearFilters}}" visible="true" stepKey="clickClearFilters1"/>
+        <waitForPageLoad time="30" stepKey="waitForPageLoad2"/>
+        <actionGroup ref="AdminGridFilterSearchResultsByInput" stepKey="filterSearchResults1">
+            <argument name="selector" value="AdminManageSourcesGridFilterControls.code"/>
+            <!--TODO Change this to scalar when MQE-498 is implemented-->
```

Comment:
This task was implemented. Please remove TODO and update test.

Reply:
Done